extensions/amp-story-page-attachment/0.1/amp-story-open-page-attachment.css

```diff
@@ -168,6 +168,10 @@
   animation: i-amphtml-tap-scale var(--i-amphtml-page-attachment-ui-animation-duration) var(--i-amphtml-page-attachment-ui-animation-delay) both !important;
 }
 
+.i-amphtml-story-page-open-attachment-outlink-no-animation-exp.i-amphtml-story-page-open-attachment-outlink-no-animation-exp[active] {
```

Comment:
If you want to avoid the animations above, it's better to do `.i-amphtml-story-page-open-attachment-outlink[active]:not(.i-amphtml-story-page-open-attachment-outlink-no-animation-exp) {animation: i-amphtml-tap-scale...}` instead of overriding it with `animation: none` on a new property. Same below.